### PR TITLE
Update pt_BR.po

### PR DESCRIPTION
**Pull request by wwweslei**
Error correction

### CODE REVIEW

File: po/pt_BR.po [hunk 1]
```diff
@@ -1048,7 +1048,7 @@ msgid ""
 "Your list of podcast subscriptions is empty. Please subscribe to some "
 "podcasts first before trying to export your subscription list."
 msgstr ""
-"Sua lista de inscrição de podcasts esta vazia, por favor se inscreva aluns "
+"Sua lista de inscrição de podcasts esta vazia, Assine por favor alguns "
```
Comment (vossad01) [inline po/pt_BR.po]: Is it grammatically correct to capitalize after a comma in Portuguese?  Should that be a period now instead?

Comment (wwweslei) [inline po/pt_BR.po]: it is not correct, i made a mistake in editing.

**Comment by vossad01** [comment]
No worries :-).  If you can push the correction, I can merge this from my phone.  Otherwise it will wait until tomorrow when I am at a computer to be able to edit then merge.

**Comment by vossad01** [comment]
Thanks @wwweslei!!

Merge notes: I don't know Portuguese, but Google Translate agrees this is a better translation and shows no vandalism.

**Comment by vossad01** [comment]
@vossad01 merged commit 8064aec0 into `gpodder:gtk3`

**Pull request merged**